dicer/mappers/transmart_loader_mapper.py

```diff
@@ -0,0 +1,69 @@
+from transmart_loader.transmart import DataCollection, Concept as TLConcept, Visit as TLVisit, TreeNode as TLTreeNode, \
+    Patient as TLPatient, Observation as TLObservation, Relation as TLRelation, Modifier as TLModifier, \
+    Dimension as TLDimension, Study as TLStudy, TrialVisit as TLTrialVisit, RelationType as TLRelationType
+
+from dicer.mappers.mapper_helper import *
+from dicer.mappers.observation_mapper import ObservationMapper
+from dicer.mappers.ontology_mapper import OntologyMapper
+from dicer.mappers.patient_mapper import PatientMapper
+from dicer.query_results import QueryResults
+from dicer.transmart import StudyDimensionElement, TrialVisitDimensionElement
+
+
+class TransmartLoaderMapper:
+    """
+    Transmart query results to transmart-loader objects mapping
+    """
+    def __init__(self):
+        self.concepts: List[TLConcept] = []
+        self.modifiers: List[TLModifier] = []
+        self.dimensions: List[TLDimension] = []
+        self.studies: List[TLStudy] = []
+        self.trial_visits: List[TLTrialVisit] = []
+        self.visits: List[TLVisit] = []
+        self.ontology: List[TLTreeNode] = []
+        self.patients: List[TLPatient] = []
+        self.observations: List[TLObservation] = []
+        self.relation_types: List[TLRelationType] = []
+        self.relations: List[TLRelation] = []
+
+    def map_query_results(self, query_results: QueryResults) -> DataCollection:
+
+        patient_mapper = PatientMapper()
+        self.patients = patient_mapper.map_patients(query_results.observations.dimensionElements.get('patient', []))
```

Comment:
Here it seems there is no use for having the lists data elements as class members. Looks like this can be just a function.